contrib/instrumentation.sh

```diff
@@ -3,3 +3,6 @@
 adb logcat -c
 adb logcat *:E -v color &
 ./gradlew jacocoInstrumentationTestReport
+adb shell screencap -p /sdcard/screencap.png
```

Comment:
it looks like it is possible to skip writing it to the sdcard, [link](https://stackoverflow.com/a/37191719/12225251)

Reply:
Thanks, I'll try this out.